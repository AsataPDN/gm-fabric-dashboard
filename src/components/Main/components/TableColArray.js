import TableCol from "./TableCol";

const TableColArray = TableCol.extend`
  text-align: right;
  span + span {
    font-size: 80%;
    opacity: 0.8;
  }
`;

export default TableColArray;
