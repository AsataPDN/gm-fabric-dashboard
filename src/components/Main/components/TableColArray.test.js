import React from "react";
import { shallow } from "enzyme";
import TableColArray from "./TableColArray";

describe("TableColArray", () => {
  it("matches snapshot", () => {
    const aTableColArray = shallow(<TableColArray />);
    expect(aTableColArray).toMatchSnapshot();
  });
});
