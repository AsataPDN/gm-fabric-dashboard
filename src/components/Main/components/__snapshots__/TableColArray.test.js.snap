// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableColArray matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1 1 12%;
  -ms-flex: 1 1 12%;
  flex: 1 1 12%;
  text-align: left;
  text-align: right;
}

.c0 span + span {
  font-size: 80%;
  opacity: 0.8;
}

<div
  className="c0"
/>
`;
